src/main/java/randoop/main/Minimize.java

```diff
@@ -245,9 +242,10 @@ public Boolean call() throws Exception {
    * @param verboseOutput whether to produce verbose output
    * @return true if minimization produced a (possibly unchanged) file that fails the same way as
    *     the original file
+   * @throws IOException thrown if write to file fails
```

Comment:
Remove "thrown"; the first word after the exception name is often "if".
The same comment applies to other `@throws` clauses.